utils/build/docker/java/spring-boot/src/main/java/com/datadoghq/system_tests/springboot/AppSecIast.java

```diff
@@ -314,12 +314,14 @@ void secureXSS(final ServletResponse response) throws IOException {
 
     @GetMapping(value = "/hstsmissing/test_insecure", produces = "text/html")
     public String hstsHeaderMissingInsecure(HttpServletResponse response) {
+        response.addHeader("X-Content-Type-Options", "nosniff");
```

Comment:
This line will re-introduce the flakyness, no ? see #1666 
cc @manuel-alvarez-alvarez

Reply:
removed

Reply:
I don't see it removed, and it's failing : https://github.com/DataDog/system-tests/actions/runs/6480588115/job/17596485817?pr=1657